cmd/cmd_ops.go

```diff
@@ -0,0 +1,45 @@
+//
+// DISCLAIMER
+//
+// Copyright 2016-2022 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+
+package cmd
+
+import (
+	goflag "flag"
+
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+)
+
+var (
+	cmdOps = cobra.Command{
+		Use: "arangodb_ops",
```

Comment:
Not sure why it is "arangodb_ops" and not just "ops". With "arangodb_ops" the command would look self-repeatable like `./arangodb_operator_ops arangodb_ops ...`